### DISCUSSION TITLE
Wondering which wox I should use

### DISCUSSION THREAD

**Discussion by CookieSalesman** [Wox-launcher/Wox#4045] [Q&A]
Hello. Not a very tech savvy user. Just want to ask the community without bothering the developer. I have Wox v1 and v2 installed. I remember I was going to uninstall v2 because something wasn't right. Does anyone know what I should do? Is v2 jut better in every way?

**Comment by qianlifeng** [comment]
Pls refer [Uninstall](https://wox-launcher.github.io/Wox/#/uninstall) doc to uninstall v2

Currently,  v2 is not better than v1 on Windows. But v1 has stopped maintenance, so choosing V2 is a better option, as it will continue to receive updates and bug fixes.

**Reply by CookieSalesman** [reply]
thanks a lot

**Comment by CookieSalesman** [comment]
Would like to ask... does v2 have a settings menu? I cannot seem to access it.

**Reply by qianlifeng** [reply]
Search `wox setting` in the query box

**Reply by CookieSalesman** [reply]
I'm not sure if it worked
![image](https://github.com/Wox-launcher/Wox/assets/9441933/46411dde-6173-48d1-89ba-94f0a55bd027)

**Reply by qianlifeng** [reply]
It's not Wox v2

**Reply by CookieSalesman** [reply]
But this is where I got it from: https://github.com/Wox-launcher/Wox/releases/tag/v2.0.0-nightly
